packages/calling-stateful-client/src/CallSubscriber.ts

```diff
@@ -219,6 +219,18 @@ export class CallSubscriber {
     this._breakoutRoomsSubscriber.unsubscribe();
   };
 
+  // This is a helper function to safely call subscriber functions. This is needed in order to prevent events
+  // with the same event type from failing to fire due to a subscriber throwing an error upon subscription.
+  // Wrap your listeners with this helper function if your listener can fail due to initialization or fail
+  // during a function call. This will prevent other events using the same event type from failing to fire.
+  private _safeSubscribe(subscriber: () => void): void {
+    try {
+      subscriber();
+    } catch (e) {
+      this._context.teeErrorToState(e as Error, 'CallAgent.feature');
```

Comment:
I am using CallAgent.feature. 
If we wanted to add a new one like `CallSubscriber.safeSubscribe` let me know. Then I will also need to add it into inferreredCallErrorTarget
![image](https://github.com/user-attachments/assets/4b6f66d0-f421-4770-a42c-da10f2ce69b2)

Reply:
Would require api changes as it also needs to be added to CallErrorTarget.
I am hesitant to add one specifically for CallSubscriber as it is not an exact error from the Calling SDK. More so a feature error from the Calling sdk. But let me know otherwise .

Reply:
I think having this throw the CallAgent.feature is the right thing here. since we are technically running into this issue due to the initialization of a Calling feature.

Reply:
It's not quite correct because the error isn't necessarily from feature.. do we need to tee it to state or should we just console error it?

Reply:
Updated it to 'Call.on'